notebooks/working/5.0-mab-simulation-causal-graph.md

```diff
@@ -801,6 +802,24 @@ def FitAlternativeRegression(regressions, reg_types, data):
             
             # Store model
             regression_results[regressions[reg][1]+'_on_'+regressions[reg][0]] = results
+
+        # Multinomial logistic regression
+        if reg_types[reg] == 'multinomial_logistic':
+            # Prepare data and initialize model
+            # regressions[reg][0] COULD be a list
+            data_x = data[regressions[reg][0]]
```

Comment:
If `regressions[reg][0]` is a string, will line 815 work? I was of the impression that sklearn estimators need a 2d array for `X`. If so you should add a line that converts `data_x` to be 2D if necessary.

Reply:
Whoops! totally missed that. Thanks for catching that! I just made the change, check it out.